packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -12,51 +12,38 @@ import AccountTransferReceipt from './account-transfer-receipt';
 import AccountTransferForm from './account-transfer-form';
 import AccountTransferNoAccount from './account-transfer-no-account';
 import AccountTransferLocked from './account-transfer-locked';
+import { useStore } from '../../hooks';
 
 type TAccountTransferProps = {
-    accounts_list: Array<TAccountsList>;
-    container: string;
-    error: {
-        is_show_full_page: boolean;
-        message: string;
-    };
-    has_no_account: boolean;
-    has_no_accounts_balance: boolean;
-    is_cashier_locked: boolean;
-    is_crypto_transactions_visible: boolean;
-    is_loading: boolean;
-    is_switching: TClientStore['is_switching'];
-    is_transfer_confirm: boolean;
-    is_transfer_locked: boolean;
-    is_virtual: TClientStore['is_virtual'];
-    onMount: () => void;
-    recentTransactionOnMount: () => void;
-    setAccountTransferAmount: (amount: number | string) => void;
-    setActiveTab: (container: string) => void;
-    setIsTransferConfirm: (status: boolean) => void;
     setSideNotes: (notes: TSideNotesProps) => void;
 };
 
-const AccountTransfer = ({
-    accounts_list,
-    container,
-    error,
-    has_no_account,
-    has_no_accounts_balance,
-    is_cashier_locked,
-    is_crypto_transactions_visible,
-    is_loading,
-    is_switching,
-    is_transfer_confirm,
-    is_transfer_locked,
-    is_virtual,
-    onMount,
-    recentTransactionOnMount,
-    setAccountTransferAmount,
-    setActiveTab,
-    setIsTransferConfirm,
-    setSideNotes,
-}: TAccountTransferProps) => {
+const AccountTransfer = ({ setSideNotes }: TAccountTransferProps) => {
+    const {
+        modules: {
+            cashier: { account_transfer, general_store, transaction_history },
+        },
+        client,
+    } = useStore();
+
+    const {
+        accounts_list,
+        container,
+        error,
+        has_no_account,
+        has_no_accounts_balance,
+        is_transfer_confirm,
+        onMountAccountTransfer: onMount,
+        setAccountTransferAmount,
+        setIsTransferConfirm,
+    } = account_transfer;
+
+    const { is_cashier_locked, is_loading, is_transfer_locked, setActiveTab } = general_store;
```

Comment:
@vinu-deriv Seems like `is_transfer_locked` doesn’t exist on `general_store` 🤔 Not sure how it was working before 😅

Reply:
yes, I had same query while working on the file, I taught I will fix in other card. Anyway I have made the correction will push it in next changes